src/main/java/techcourse/myblog/domain/Article.java

```diff
@@ -1,4 +1,75 @@
 package techcourse.myblog.domain;
 
+import java.util.Objects;
+
 public class Article {
+    private static final int EMPTY_ID = -1;
+
+    private final int id;
+    private final String title;
+    private final String coverUrl;
+    private final String contents;
+
+    public Article(int id, String title, String coverUrl, String contents) {
+        this.id = id;
+        this.title = title;
+        this.coverUrl = coverUrl;
+        this.contents = contents;
+    }
+
+    public static Article of(String title, String coverUrl, String contents) {
```

Comment:
생성자가 있는데 팩터리 메서드를 따로 만들 필요가 있을까요?

Reply:
현재 id 없이 생성되는 부분과 id를 알고 생성하는 부분이 다릅니다. 

일단 역할을 메서드 명에서 명시적으로 표현해보았습니다. (정적 팩터리 메소드의 장점 중 하나가.. 이름으로 의미를 표현할 수 있는 거라고 생각합니다!)

```suggestion
    public static Article createWithoutId(String title, String coverUrl, String contents) {
```